server/server_test.go

```diff
@@ -7751,6 +7756,7 @@ stream
 		.message('message')
 		.details('details')
 		.crit(lambda: TRUE)
+		.sensu()
```

Comment:
It looks like I am getting this error in services/sensu/sensutest/sensutest.go line 53

"accept tcp 127.0.0.1:48578: use of closed network connection"

Not sure what I changed that would cause this to happen. Any suggestions on how to fix the issue?